ui/static/ui/css/mit-lore.css

```diff
@@ -462,11 +462,12 @@ footer {
 }
 
 .drawer-tabs li a {
-    border: 0 !important;
+    border-radius:0 !important;
```

Comment:
Can you add a space in front of all `:` for consistency with the rest of the file?